nailgun/entities.py

```diff
@@ -2875,6 +2875,99 @@ def read(self, entity=None, attrs=None, ignore=()):
         return super(Repository, self).read(entity, attrs, ignore)
 
 
+class RHCIDeployment(
+        Entity, EntityCreateMixin, EntityDeleteMixin, EntityReadMixin):
+    """A representation of a RHCI deployment entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'name': entity_fields.StringField(required=True),
+            'organization_id': entity_fields.IntegerField(required=True),
+            'lifecycle_environment_id': entity_fields.IntegerField(
+                required=True),
+            'deploy_rhev': entity_fields.BooleanField(required=True),
+            'rhev_engine_host_id': entity_fields.IntegerField(required=True),
+            'rhev_storage_type': entity_fields.StringField(required=True),
+            'rhev_engine_admin_password': entity_fields.StringField(),
+        }
+        self._meta = {
+            'api_path': 'fusor/api/v21/deployments',
+            'server_modes': ('sat'),
+        }
+        super(RHCIDeployment, self).__init__(server_config, **kwargs)
+
+    def path(self, which=None):
+        """Extend ``nailgun.entity_mixins.Entity.path``.
+
+        The format of the returned path depends on the value of ``which``:
+
+        deploy
+            /deployments/<id>/deploy
+
+        ``super`` is called otherwise.
+
+        """
+        if which in ('deploy'):
```

Comment:
I believe `make lint` is complaining about the unnecessary parentheses here. `if which == 'deploy':` should satisfy it.